addons/stock_calendar/tests/test_stock_calendar.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, timedelta
+
+from odoo.tests import common
+from odoo import fields
+
+
+class TestsStockCalendar(common.TransactionCase):
+
+    def setUp(self):
+        super(TestsStockCalendar, self).setUp()
+
+        # Create a partner
+        self.stock_warehouse0_id = self.ref('stock.warehouse0')
+        self.purchase_route_warehouse0_buy_id = self.ref('purchase.route_warehouse0_buy')
+        self.stock_picking_type_out_id = self.ref('stock.picking_type_out')
+        self.stock_location_id = self.ref('stock.stock_location_stock')
+        self.stock_location_customer_id = self.ref('stock.stock_location_customers')
+
+        self.ResPartner = self.env['res.partner'].create({
+            'name': 'Supplier',
+            'supplier': 1
+        })
```

Comment:
Waw Waw Waw please...
Create method return created recordset so please variable name should be in snake case.